helm/kubeblocks/falkordb-cluster-omnistrate/tests/e2e_omnistrate/test_omnistrate_replication.py

```
@@ -0,0 +1,476 @@
+        eps = instance.get_connection_endpoints()
+        nodes = sorted(
+            [e for e in eps if e["id"].startswith("node-")], key=lambda x: x["id"]
+        )
+        
+        # Perform multiple failovers
+        for i, node in enumerate(nodes[:2]):
+            logging.info(f"Failover {i+1}: Triggering on node {node['id']}")
+            
+            def do_failover():
+                instance.trigger_failover(
+                    replica_id=node["id"],
```

Ruff (0.14.10)

Function definition does not bind loop variable `node`

(B023)